extras/Build/CMake/JUCEUtils.cmake

```diff
@@ -1337,6 +1347,40 @@ function(_juce_set_plugin_target_properties shared_code_target kind)
         endif()
 
         _juce_copy_after_build(${shared_code_target} ${target_name} "${output_path}" JUCE_VST3_COPY_DIR)
+    elseif(kind STREQUAL "LV2")
+        set_target_properties(${target_name} PROPERTIES
+            BUNDLE_EXTENSION lv2
+            PREFIX ""
+            SUFFIX .lv2
+            BUNDLE TRUE
+            XCODE_ATTRIBUTE_WRAPPER_EXTENSION lv2
+            XCODE_ATTRIBUTE_LIBRARY_STYLE Bundle
+            XCODE_ATTRIBUTE_GENERATE_PKGINFO_FILE YES)
+
+        _juce_create_windows_package(${shared_code_target} ${target_name} lv2 "" x86-win x86_64-win)
```

Comment:
Similar to the macOS target properties, I don't think we'll need this if we only target Linux.

And if we _do_ want to target Windows, do we really want to call this function? It seems to create a bundle containing a desktop `.ini` and an icon file, and I don't think that's really necessary if all we need to do is put the library and the .ttl files in one folder.

Reply:
Yeah, I'm not really sure how necessary this is... but then again, I also don't understand why JUCE did this for VST3?

Reply:
Yeah, I'd also like to know why they did it this way for VST3, and what the preferred way of bundling an LV2 plugin looks like for Windows. On Linux it's really just as simple as a folder containing everything.